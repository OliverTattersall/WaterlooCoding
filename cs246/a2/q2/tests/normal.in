create a +infinity
create b 0
create c 15
create d 20
* c d 
* d c 
* b d 
* b c 
- d c 
+ d c 
* a c 
* a d 
u- d 
u- c 
u- b
