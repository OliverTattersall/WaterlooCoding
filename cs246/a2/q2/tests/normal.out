300
300
0
0
5
35
+infinity
+infinity
-20
-15
0
